1/22/2018 11:49:30 PM: [ln:822] Formulario: FTPUpload() - Excepcion. Cargando archivo de boceto por FTP. ERROR: Specified argument was out of the range of valid values.
Parameter name: value.
1/22/2018 11:49:30 PM: [ln:823] Formulario: FTPUpload() - Excepcion. URL: ftp://ftp.site4now.net:21/Cartelux/cxpedidos/Resources/Bocetos/potty.png.
1/23/2018 12:52:45 PM: [ln:951] Formulario: FTPUpload_2() - Excepcion. Cargando archivo de boceto por FTP. ERROR: mammanostra-delivery-premium.jpg.
1/23/2018 12:52:45 PM: [ln:952] Formulario: FTPUpload_2() - Excepcion. URL: .
1/23/2018 12:52:52 PM: [ln:951] Formulario: FTPUpload_2() - Excepcion. Cargando archivo de boceto por FTP. ERROR: mammanostra-delivery-premium.jpg.
1/23/2018 12:52:52 PM: [ln:952] Formulario: FTPUpload_2() - Excepcion. URL: .
1/23/2018 12:55:30 PM: [ln:951] Formulario: FTPUpload_2() - Excepcion. Cargando archivo de boceto por FTP. ERROR: CX-web1.png.
1/23/2018 12:55:30 PM: [ln:952] Formulario: FTPUpload_2() - Excepcion. URL: .
1/23/2018 12:55:37 PM: [ln:951] Formulario: FTPUpload_2() - Excepcion. Cargando archivo de boceto por FTP. ERROR: CX-web1.png.
1/23/2018 12:55:37 PM: [ln:952] Formulario: FTPUpload_2() - Excepcion. URL: .
1/23/2018 1:37:24 PM: [ln:952] Formulario: FTPUpload_2() - Excepcion. Cargando archivo de boceto por FTP. ERROR: mammanostra-delivery-premium.jpg.
1/23/2018 1:37:25 PM: [ln:953] Formulario: FTPUpload_2() - Excepcion. URL: .
1/23/2018 1:40:34 PM: [ln:952] Formulario: FTPUpload_2() - Excepcion. Cargando archivo de boceto por FTP. ERROR: mammanostra-delivery-premium.jpg.
1/23/2018 1:40:34 PM: [ln:953] Formulario: FTPUpload_2() - Excepcion. URL: .
1/24/2018 4:31:11 AM: [ln:1143] Formulario: UploadMegaAPI() - Excepcion. Copiando archivo al server y guardando en BD. ERROR: Sequence contains more than one matching element.
1/24/2018 4:31:11 AM: [ln:1144] Formulario: UploadMegaAPI() - Excepcion. URL: TRABAJOS/.
1/24/2018 4:31:18 AM: [ln:1143] Formulario: UploadMegaAPI() - Excepcion. Copiando archivo al server y guardando en BD. ERROR: Sequence contains more than one matching element.
1/24/2018 4:31:18 AM: [ln:1144] Formulario: UploadMegaAPI() - Excepcion. URL: TRABAJOS/.
1/24/2018 4:34:16 AM: [ln:1143] Formulario: UploadMegaAPI() - Excepcion. Copiando archivo al server y guardando en BD. ERROR: Sequence contains more than one matching element.
1/24/2018 4:35:07 AM: [ln:1144] Formulario: UploadMegaAPI() - Excepcion. URL: TRABAJOS/.
1/24/2018 4:35:15 AM: [ln:1143] Formulario: UploadMegaAPI() - Excepcion. Copiando archivo al server y guardando en BD. ERROR: Sequence contains more than one matching element.
1/24/2018 4:35:15 AM: [ln:1144] Formulario: UploadMegaAPI() - Excepcion. URL: TRABAJOS/.
1/24/2018 4:37:01 AM: [ln:1143] Formulario: UploadMegaAPI() - Excepcion. Copiando archivo al server y guardando en BD. ERROR: Sequence contains more than one matching element.
1/24/2018 4:37:01 AM: [ln:1144] Formulario: UploadMegaAPI() - Excepcion. URL: TRABAJOS/.
1/24/2018 4:37:08 AM: [ln:1143] Formulario: UploadMegaAPI() - Excepcion. Copiando archivo al server y guardando en BD. ERROR: Sequence contains more than one matching element.
1/24/2018 4:37:08 AM: [ln:1144] Formulario: UploadMegaAPI() - Excepcion. URL: TRABAJOS/.
1/25/2018 12:26:13 AM: [ln:1136] Formulario: UploadMegaAPI() - Excepcion. Copiando archivo al server y guardando en BD. ERROR: One or more errors occurred..
1/25/2018 12:26:13 AM: [ln:1137] Formulario: UploadMegaAPI() - Excepcion. URL: .
1/25/2018 12:26:28 AM: [ln:1136] Formulario: UploadMegaAPI() - Excepcion. Copiando archivo al server y guardando en BD. ERROR: Sequence contains more than one matching element.
1/25/2018 12:26:28 AM: [ln:1137] Formulario: UploadMegaAPI() - Excepcion. URL: TRABAJOS/.
1/25/2018 12:29:02 AM: [ln:1136] Formulario: UploadMegaAPI() - Excepcion. Copiando archivo al server y guardando en BD. ERROR: Sequence contains more than one matching element.
1/25/2018 12:29:02 AM: [ln:1137] Formulario: UploadMegaAPI() - Excepcion. URL: TRABAJOS/.
1/25/2018 12:29:12 AM: [ln:1136] Formulario: UploadMegaAPI() - Excepcion. Copiando archivo al server y guardando en BD. ERROR: Sequence contains more than one matching element.
1/25/2018 12:29:12 AM: [ln:1137] Formulario: UploadMegaAPI() - Excepcion. URL: TRABAJOS/.
2/8/2018 12:32:14 AM: [ln:74] Acceso: Perform_login() - Excepcion. Haciendo login. ERROR: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server).
2/8/2018 12:33:22 AM: [ln:74] Acceso: Perform_login() - Excepcion. Haciendo login. ERROR: An error occurred while executing the command definition. See the inner exception for details..
2/8/2018 12:33:29 AM: [ln:74] Acceso: Perform_login() - Excepcion. Haciendo login. ERROR: An error occurred while executing the command definition. See the inner exception for details..
2/8/2018 12:33:52 AM: [ln:74] Acceso: Perform_login() - Excepcion. Haciendo login. ERROR: An error occurred while executing the command definition. See the inner exception for details..
2/8/2018 12:33:58 AM: [ln:74] Acceso: Perform_login() - Excepcion. Haciendo login. ERROR: An error occurred while executing the command definition. See the inner exception for details..
2/8/2018 12:35:13 AM: [ln:74] Acceso: Perform_login() - Excepcion. Haciendo login. ERROR: An error occurred while executing the command definition. See the inner exception for details..
2/11/2018 6:52:43 PM: [ln:419] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 02/20/2018.
2/11/2018 6:52:43 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 6:52:43 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 7:04:29 PM: [ln:419] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 02/17/2018.
2/11/2018 7:04:44 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 7:05:32 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 7:07:47 PM: [ln:419] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 02/14/2018.
2/11/2018 7:09:48 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 7:09:50 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 7:11:25 PM: [ln:419] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 02/15/2018.
2/11/2018 7:11:25 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 7:11:25 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 7:13:05 PM: [ln:419] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 02/14/2018.
2/11/2018 7:13:05 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 7:13:05 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 7:14:21 PM: [ln:419] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 02/22/2018.
2/11/2018 7:14:21 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 7:14:21 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/11/2018 7:19:54 PM: [ln:419] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 2/15/2018 12:00:00 AM.
2/11/2018 7:19:54 PM: [ln:440] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/12/2018 7:02:49 AM: [ln:490] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 2/13/2018 12:00:00 AM.
2/12/2018 7:02:49 AM: [ln:511] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/12/2018 7:03:00 AM: [ln:490] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 2/13/2018 12:00:00 AM.
2/12/2018 7:03:00 AM: [ln:511] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/12/2018 7:13:15 AM: [ln:492] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 2/13/2018 12:00:00 AM.
2/12/2018 7:13:15 AM: [ln:513] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/12/2018 7:13:28 AM: [ln:492] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 2/13/2018 12:00:00 AM.
2/12/2018 7:13:28 AM: [ln:513] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/12/2018 7:13:45 AM: [ln:492] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 2/13/2018 12:00:00 AM.
2/12/2018 7:13:45 AM: [ln:513] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/12/2018 7:13:52 AM: [ln:492] Formulario: GetDatetimeFormated() - Excepcion. Convirtiendo datetime. ERROR: 2/13/2018 12:00:00 AM.
2/12/2018 7:13:53 AM: [ln:513] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
20/02/2018 6:36:51: [ln:513] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
20/02/2018 6:36:52: [ln:513] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/23/2018 5:32:51 PM: [ln:128] Formulario: GetURLParam_Decrypted() - Excepcion. Desencriptando ID del parámetro URL. ERROR: Invalid length for a Base-64 char array or string..
2/23/2018 5:33:13 PM: [ln:128] Formulario: GetURLParam_Decrypted() - Excepcion. Desencriptando ID del parámetro URL. ERROR: Padding is invalid and cannot be removed..
2/23/2018 5:33:19 PM: [ln:128] Formulario: GetURLParam_Decrypted() - Excepcion. Desencriptando ID del parámetro URL. ERROR: Padding is invalid and cannot be removed..
2/23/2018 6:31:15 PM: [ln:513] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
2/23/2018 6:31:15 PM: [ln:513] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
3/31/2018 5:41:44 PM: [ln:503] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
3/31/2018 5:41:44 PM: [ln:503] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
3/31/2018 6:10:55 PM: [ln:504] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
3/31/2018 6:10:55 PM: [ln:504] Formulario: Guardar_Contexto() - Excepcion. Guardando en la base de datos. ERROR: An error occurred while updating the entries. See the inner exception for details..
